cmd/api/src/api/registration/v2.go

```diff
@@ -131,6 +131,8 @@ func NewV2API(cfg config.Configuration, resources v2.Resources, routerInst *rout
 		routerInst.GET("/api/v2/features", resources.GetFlags).RequirePermissions(permissions.AppReadApplicationConfiguration),
 		routerInst.PUT("/api/v2/features/{feature_id}/toggle", resources.ToggleFlag).RequirePermissions(permissions.AppWriteApplicationConfiguration),
 
+		routerInst.POST("/api/v2/database-management", resources.HandleDatabaseManagement).RequirePermissions(permissions.WipeDB),
```

Comment:
My opinion is that this endpoint should be renamed. To me "database management" implies a lot more beyond clearing data. Maybe "/api/v2/clear-database"?